src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -958,10 +957,8 @@ public static void configCheck(ConfigType type) {
 			config.addDefault("Abilities.Earth.Passive.FerroControl.Enabled", true);
 
 			config.addDefault("Abilities.Earth.Catapult.Enabled", true);
-			config.addDefault("Abilities.Earth.Catapult.Length", 6);
-			config.addDefault("Abilities.Earth.Catapult.Push", 4);
-			config.addDefault("Abilities.Earth.Catapult.ShiftModifier", 2);
-			config.addDefault("Abilities.Earth.Catapult.Cooldown", 1500);
+			config.addDefault("Abilities.Earth.Catapult.MaxDistance", 40);

```

Comment:
We need the time between tiers to be configurable as well as the separate modifiers.
Ei:
TierChargeTime: 1000 (ms)
TierPowerModifier (2) (2 times more powerful for each tier)